jenkins/jobs/capm3-e2e-tests.pipeline

```diff
@@ -1,7 +1,5 @@
-import java.text.SimpleDateFormat
```

Comment:
Was this dropped by purpose?

Reply:
yes, it's not used anywhere in this script.